.github/workflows/release-python.yml

```diff
@@ -79,75 +61,37 @@ jobs:
         # Note: the default manylinux is manylinux2014
         run: |
           python -m pip install -U pip
-          python -m pip install "cibuildwheel>=2.4,<3"
+          python -m pip install "cibuildwheel>=2.17,<3"
 
-      - name: Build wheel
-        if: ${{ matrix.buildplat[0] != 'macos-14' }}
+      - name: Build wheels
         env:
-          CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}
-          MACOS_TEST_SKIP: "*arm64"
+          CIBW_BUILD: ${{ matrix.buildplat[2] }}
```

Comment:
Should this step exclude manylinux1?

Reply:
No need to exclude it.  We have a separate manylinux1 to ensure that we have wheels that include that arch, but it is fine if we have duplicate wheels.